03 Misc/05 Commons Chunk Plugin/src/students.ts

```diff
@@ -0,0 +1,12 @@
+import {getAvg} from "./averageService";
+import * as $ from 'jquery';
+import leftPad = require('left-pad');
```

Comment:
Oh, the @types on it sucks big time. There is no way of using `from` with that. That is a typescript old type of import but there is no way of doing it in here because the types doesn't export a module.

I used this library for being super simple on the output